bluesky_queueserver/manager/tests/test_plan_queue_ops.py

```diff
@@ -0,0 +1,585 @@
+import asyncio
+import pytest
+import json
+import copy
+from bluesky_queueserver.manager.plan_queue_ops import PlanQueueOperations
+
+
+errmsg_wrong_plan_type = "Parameter 'plan' should be a dictionary"
+
+
+@pytest.fixture
+def pq():
+    pq = PlanQueueOperations()
+    asyncio.run(pq.start())
+    # Clear any pool entries
+    asyncio.run(pq.delete_pool_entries())
+    yield pq
+    # Don't leave any test entries in the pool
+    asyncio.run(pq.delete_pool_entries())
```

Comment:
Should we move this fixture to `conftest.py`? Looks like it should be the place to store the fixtures.

Reply:
I guess that if it is used only for tests in this file, it should stay in this file. I don't see it being used anywhere else at this point. But it can always be moved if needed. I think the main idea is to avoid test files importing each other.